courses/free-courses-fa_IR.md

```diff
@@ -1,8 +1,10 @@
 ### Index
 
+* [Ansible](#ansible)
 * [Blockchain](#blockchain)
 * [C, C++](#c)
 * [C#](#csharp)
+* [Ceph](#ceph)
```

Comment:
What do you think about putting is in a "storage" category to help people know what it is?